equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/Row.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+@Data
+@NoArgsConstructor
+
+public class Row{
```

Comment:
This is too generic. I would suggest a class name that actually explains what it actually is. Not how we are sorting them